pages/api/articles/[feed].js

```diff
@@ -184,12 +185,23 @@ async function articleFeedHandler(req, res) {
       });
     });
 
+    const visitor = ua(PUBLIC_GA_TRACKING_ID);
+    const params = {
+      ec: 'RSS',
+      ea: 'Subscribe',
```

Comment:
I think we can use something like `Poll`, which better describes this action.

Reply:
👌